templates/test/ci/prow-ci-version/patches/oot-credential-provider-win.yaml

```diff
@@ -4,11 +4,22 @@
     content: |
       $ErrorActionPreference = 'Stop'
 
-      echo "Use OOT credential provider"
-      mkdir C:\var\lib\kubelet\credential-provider
-      curl.exe --retry 10 --retry-delay 5 -L "https://${AZURE_STORAGE_ACCOUNT}.blob.core.windows.net/${AZURE_BLOB_CONTAINER_NAME}/${IMAGE_TAG_ACR_CREDENTIAL_PROVIDER}/azure-acr-credential-provider.exe" --output C:\var\lib\kubelet\credential-provider\acr-credential-provider.exe
-      cp C:\var\lib\kubelet\credential-provider\acr-credential-provider.exe C:\var\lib\kubelet\credential-provider\acr-credential-provider
-      curl.exe --retry 10 --retry-delay 5 -L "https://${AZURE_STORAGE_ACCOUNT}.blob.core.windows.net/${AZURE_BLOB_CONTAINER_NAME}/${IMAGE_TAG_ACR_CREDENTIAL_PROVIDER}/credential-provider-config-win.yaml" --output C:\var\lib\kubelet\credential-provider-config.yaml
+      Write-Host "Attempting to log in to Azure with managed identity"
```

Comment:
Is there a reason why the `Write-Host` log lines are only in the windows patches? I don't see the same pattern in Jack's PR

Reply:
Good eye! 
The windows patches seem to be running on Powershell. 
While `echo` also works just fine (looked it up), I put `Write-Host` to explicitly bring this distinction up that _Windows patches will use Powershell instead of shell_ to run the scripts. 

Also notice how `curl` is `curl.exe` in windows patches.